fre/pp/ppyaml_test/pp.yaml

```diff
@@ -1,225 +1,79 @@
-        interval: *ANA_AMIP_LEN
+    inputRealm: 'atmos'
   - type: "atmos_month_aer"
-    start: *ANA_AMIP_START
     sources: "atmos_month_aer"
     sourceGrid: "cubedsphere"
-    xyInterp: *defaultxyInterp
+    xyInterp: *xyInterp
     interpMethod: "conserve_order1"
-    cmip: "on"
-    timeAverage:
-      - source: "monthly"
-        interval: *PP_AMIP_CHUNK96
-      - source:  "monthly"
-        interval: *ANA_AMIP_LEN
-    timeSeries:
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
+    inputRealm: 'atmos'
   - type: "atmos_diurnal"
-    start: *ANA_AMIP_START
     sources: "atmos_diurnal"
-    sourceGrid: "cubed-sphere"
-    xyInterp: *defaultxyInterp
-    interpMethod: conserve_order2
-    timeSeries:
-      - freq: "monthly"
-        chunklength: *PP_AMIP_CHUNK96
+    sourceGrid: "cubedsphere"
+    xyInterp: *xyInterp
+    interpMethod: "conserve_order2"
+    inputRealm: 'atmos'
   - type: "atmos_scalar"
-    start: *ANA_AMIP_START
     sources: "atmos_scalar"
-    cmip: "on"
-    timeSeries:
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
-      - freq: "monthly"
-        source: "atmos_global_cmip"
-        chunkLength: *PP_AMIP_CHUNK96
-  - type: "tracer_level"
-    start: *ANA_AMIP_START
-    sources: "atmos_tracer"
-    sourceGrid: "cubedsphere"
-    xyInterp: *defaultxyInterp
-    interpMethod: "conserve_order1"
-    cmip: "on"
-    timeSeries:
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
-      - freq: "annual"
-        chunkLength: *PP_AMIP_CHUNK96
-    timeAverage:
-      - source: "monthly"
-        interval: *PP_AMIP_CHUNK96
-      - source: "monthly"
-        interval: *ANA_AMIP_LEN
   - type: "aerosol_cmip"
-    start: *ANA_AMIP_START
-    xyInterp: *defaultxyInterp
+    xyInterp: *xyInterp
     sources: "aerosol_month_cmip"
     sourceGrid: "cubedsphere"
     interpMethod: "conserve_order1"
-    cmip: "on"
-    timeSeries:
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
-        source: "aerosol_month_refined"
+    inputRealm: 'atmos'
   - type: "land"
-    start: *ANA_AMIP_START
     sources: "land_month"
     sourceGrid: "cubedsphere"
-    xyInterp: *defaultxyInterp
+    xyInterp: *xyInterp
     interpMethod: "conserve_order1"
-    cmip: "off"
-    timeSeries:
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
-    timeAverage:
-      - source: "monthly"
-        interval: *PP_AMIP_CHUNK96
-      - source: "monthly"
-        interval: *ANA_AMIP_LEN
+    inputRealm: 'land'
   - type: "land_cmip"
-    start: *ANA_AMIP_START
     sources: "land_month_cmip"
     sourceGrid: "cubedsphere"
-    xyInterp: *defaultxyInterp
+    xyInterp: *xyInterp
     interpMethod: "conserve_order1"
-    cmip: "off"
-    timeSeries:
-      - freq: "monthly"
-        chunkLength: *PP_AMIP_CHUNK96
-
-tmpdir:
-  tmpdirpath:
-
-install-exp-script:
-    - path: 
-    - install-option:
-        install: 
```

Comment:
I wasn't sure if we still wanted these or not. Currently, they are not being used in the configure yaml script, but they are useful for running on `generic` sites (non-container use: stage-history used tmp path and the `install-option`  adds on a `symlink-dirs ...` to the cylc install command, if the user wanted the cylc-run directory to be put in a location with more space). 

Just a clarification, should we keep these for a non-container use on other sites? (and then I guess add onto the configure yaml script eventually)

Reply:
Thanks! I confess I didn't really study how these were used and had a feeling I was throwing away too much.

I'll add those back in, no problem. But I'm still not sure how they're used.. did I remove too much from configure_script_yaml.py ? Or are they used by a container specific tool?

Reply:
No, you didn't get rid of too much regarding these! The configure script didn't actually use these yet.... They were used in the old `apply-edits` script I had but when we revamped that to be configure_script_yaml, I forgot to add these back in. I can probably work on adding the functionality for these in if we want. 

They were for a non-container use. For example, if one was on niagara:
- stage-history needed a `tmp` location set (this wasn't an issue on ppan because somewhere in environment things, it was set)
- then the cylc-run directory was taking up too much space (as the default installation was `home`; disk quota exceeded), so that install option would be populated with `--symlink-dirs........` (forgot the whole syntax at the moment) to point the cylc-run directory to be installed elsewhere (`/collab1` on niagara)

Reply:
I added them back. If they're at all useful, let's keep them for sure.

Reply:
Definitely, let's keep them. I added them back in.